solutions/USACO jan16-subseq-summing-to-sevens.mdx

```diff
@@ -5,7 +5,7 @@ title: Subsequences Summing to Sevens
 author: Óscar Garries
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_div7_silver_jan16.html)
+[Official Analysis](http://www.usaco.org/currrent/data/sol_div7_silver_jan16.html)
```

Comment:
typo: "currrent" should be "current"

Reply:
thanks! can't believe I missed that

Reply:
> thanks! can't believe I missed that

please fix this (and also update the author)